sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumIO.java

```diff
@@ -47,6 +47,13 @@
  *
  * <h3>Quick Overview</h3>
  * <p>This class lets Beam users connect to their existing Debezium implementations in an easy way.</p>
+ * <p>Currently the supported Debezium Connectors are:</p>
```

Comment:
I'd rephrase this to: "Any Kafka connector supported by Debezium should work fine with this IO. The following connectors were tested and worked well in some simple scenarios:"

Reply:
Great, changing that...